README.md

```diff
@@ -1,2 +1,12 @@
-# practice-with-wes
-learning the things
+# Practice with Wes!
+
+Learning the things! This is a general scratchpad location for Wes and Chris to share scripts and general knowledge.
+
+## Goals
+
+* Learn markdown.
+* Learn Git.
+* Learn Git workflows, e.g. issue tracking, PRs, mergers, etc.
```

Comment:
Can you review this to elaborate @necrux

Reply:
In this we will open some GitHub issues and practice creating new branches to address those issues; performing code review, and merging the code.